### PR TITLE
(6.0) Update release table to 6.0.0

### PR DESCRIPTION
**Pull request by r0mant**
The release download table embedded in some pages was still using rc.1 release, update it to 6.0.0.

I manually tested all the links and they seemed to work.

### CODE REVIEW

File: docs/pages/preview/releases-table.mdx [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
+| Linux 32-bit | [SHA256](https://get.gravitational.com/teleport-v6.0.0-linux-386-bin.tar.gz.sha256) | [teleport-v6.0.0-linux-386-bin.tar.gz](https://get.gravitational.com/teleport-v6.0.0-linux-386-bin.tar.gz) |
+| Linux 64-bit | [SHA256](https://get.gravitational.com/teleport-v6.0.0-linux-amd64-bin.tar.gz.sha256) | [teleport-v6.0.0-linux-amd64-bin.tar.gz](https://get.gravitational.com/teleport-v6.0.0-linux-amd64-bin.tar.gz) |
+| Linux 64-bit (RHEL/CentOS 6.x compatible) | [SHA256](https://get.gravitational.com/teleport-v6.0.0-linux-amd64-centos6-bin.tar.gz.sha256) | [teleport-v6.0.0-linux-amd64-centos6-bin.tar.gz](https://get.gravitational.com/teleport-v6.0.0-linux-amd64-centos6-bin.tar.gz) |
+| Linux ARMv7 | [SHA256](https://get.gravitational.com/teleport-v6.0.0-linux-arm-bin.tar.gz.sha256) | [teleport-v6.0.0-linux-arm-bin.tar.gz](https://get.gravitational.com/teleport-v6.0.0-linux-arm-bin.tar.gz) |
+| Linux 64-bit DEB | [SHA256](https://get.gravitational.com/teleport_6.0.0_amd64.deb.sha256) | [teleport_v6.0.0_amd64.deb](https://get.gravitational.com/teleport_6.0.0_amd64.deb) |
```
Comment (klizhentas) [inline docs/pages/preview/releases-table.mdx:9]: can you use `{{version}}` var?

**Review by klizhentas — CHANGES_REQUESTED** [review]

File: docs/pages/preview/releases-table.mdx (hunk 1 above)
Comment (r0mant) [reply docs/pages/preview/releases-table.mdx]: Nice, I didn't know we had it. Updated.

**Comment by klizhentas** [comment]
@r0mant please also forward-port the PR

**Comment by r0mant** [comment]
@klizhentas https://github.com/gravitational/teleport/pull/5851

**Comment by webvictim** [comment]
I'm curious why we're embedding a second set of links here, shouldn't we just be directing people off to https://goteleport.com/teleport/download? I could understand it during preview to make sure people got the correct version but now it's GA, the second set of links should probably go.

**Comment by r0mant** [comment]
@webvictim Since these are "guides", I think the idea is to let people do as much as possible without leaving the page and not disrupt the flow. I'm not a fan of duplicating the links myself to be honest but I'm not aware of a decent (or any really) way to just embed our downloads table in the docs, which would probably be ideal. Maybe that's something that's on the grv/next roadmap? As a next best thing I think we could add a note here something along the lines of "or go to our Downloads page to pick an appropriate version" or something, in addition to the table?

@klizhentas Curious what you think as well ^

**Comment by klizhentas** [comment]
I agree with @webvictim please point to downloads page

**Comment by r0mant** [comment]
@klizhentas Done.

**Review by webvictim — APPROVED** [review]

**Review by klizhentas — APPROVED** [review]

**Pull request merged**